src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Loader } from '../Loader/Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const { slug = '' } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    const getPeopleFromTheServer = async () => {
+      try {
+        const peopleFromTheServer = await getPeople();
+
+        setPeople(peopleFromTheServer);
+      } catch {
+        setIsError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    getPeopleFromTheServer();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && <Loader />}
+
+          {isError && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+
+          {(!people.length && !isLoading && !isError) && (
+            <p data-cy="noPeopleMessage">
+              There are no people on the server
+            </p>
+          )}
+
+          {people.length
```

Comment:
Better to wrap it to boolean or compare with come value, now 0 is rendered  if length is 0
![Screenshot 2023-04-18 at 19 33 31](https://user-images.githubusercontent.com/84275496/232845410-6dd3b54f-2b9a-4619-8d10-028cd5ff27f4.png)